pkgs/applications/misc/roxterm/default.nix

```diff
@@ -1,54 +1,38 @@
-{ stdenv, fetchurl, docbook_xsl, dbus, dbus-glib, expat
-, gsettings-desktop-schemas, gdk-pixbuf, gtk3, hicolor-icon-theme
-, imagemagick, itstool, librsvg, libtool, libxslt, makeWrapper
-, pkgconfig, python, pythonPackages, vte
-, wrapGAppsHook}:
+{ at-spi2-core, cmake, dbus, dbus-glib, docbook_xsl, epoxy, fetchpatch, fetchurl
+, glib, gtk3, harfbuzz, hicolor-icon-theme, libXdmcp, libXtst, libpthreadstubs
+, libselinux, libsepol, libtasn1, libxkbcommon, libxslt, p11-kit, pcre
+, pkgconfig, stdenv, utillinuxMinimal, vte, wrapGAppsHook, xmlto
+}:
 
-# TODO: Still getting following warning.
-# WARNING **: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files
-# Seems related to this:
-# https://forums.gentoo.org/viewtopic-t-947210-start-0.html
-
-let version = "3.3.2";
-in stdenv.mkDerivation rec {
-  name = "roxterm-${version}";
+stdenv.mkDerivation rec {
+  pname = "roxterm";
+  version = "3.7.5";
 
   src = fetchurl {
-    url = "mirror://sourceforge/roxterm/${name}.tar.xz";
-    sha256 = "0vjh7k4jm4bd01j88w9bmvq27zqsajjzy131fpi81zkii5lisl1k";
+    url = "https://github.com/realh/roxterm/archive/${version}.tar.gz";
```

Comment:
Do you mind adding a reference for changing the code source? Quickly looking at the website doesn't prove that this repository is related to the upstream.

Reply:
The author Tony Houghton links to that repo in https://sourceforge.net/p/roxterm/discussion/422638/thread/60da6975/?limit=25#3fc9